internal/subscription/validation.go

```diff
@@ -370,6 +383,24 @@ func validateDeadLetterPolicy(_ context.Context, m *Model, req *metrov1.Subscrip
 	return nil
 }
 
+func validatePushEndpoint(_ context.Context, req *metrov1.Subscription) error {
+	if req.GetPushConfig() != nil {
+		u, err := url.Parse(req.GetPushConfig().PushEndpoint)
+		if err != nil {
+			return ErrInvalidPushEndpointURL
+		}
+
+		// eg: DialTimeout("tcp", "golang.org:http", timeout)
+		networkAddress := u.Hostname() + ":" + u.Scheme
```

Comment:
Shouldn't the format be `Scheme : Hostname` ?

Reply:
Say `https://dummy.rzp.in` ?

Reply:
No the other way 
```

/ Examples:
//	Dial("tcp", "golang.org:http")
//	Dial("tcp", "192.0.2.1:http")
//	Dial("tcp", "198.51.100.1:80")
//	Dial("udp", "[2001:db8::1]:domain")
//	Dial("udp", "[fe80::1%lo0]:53")
//	Dial("tcp", ":80")
```